File: Maths_And_Stats/Series/Relative_Primes/RelativePrimes.php
<?php

// This will return their greatest common divisor (gcd)
function gcd($int1, $int2) {
    if ($int1 == 0 || $int2 == 0) return 0;
    if ($int1 == $int2) return $int1;
    if ($int1 > $int2) return gcd($int1 - $int2, $int2);
    return gcd($int1, $int2 - $int1);
}

// check if they are both relatively prime or not
function isRelativelyPrime($int1, $int2) {
    $result = gcd($int1, $int2) == 1 ? "is relatively prime \n": "is not relatively prime \n";
    return $result;
}

echo isRelativelyPrime(4, 9);
echo isRelativelyPrime(6, 8);

File: Maths_And_Stats/Series/Relative_Primes/RelativePrimes.c
#include<stdio.h>

void main(){
        int p,i,j;
        int remainder = 2;
        int divident,divisor;

        printf("Enter Number\n");
        scanf("%d",&p);

        for(i = 2 ; i < p ; i++){

                divident  = p;
                divisor = i;

                while(divisor != 0){

                        remainder = divident % divisor;
                        divident  = divisor;
                        divisor  = remainder;
                }
       
                if(divident  == 1){
                        printf("Relatively Prime Number is : %d \n" ,i);
                }
        }

}

/* Output:
Enter Number 
8
Relatively Prime Number is : 3
Relatively Prime Number is : 5
Relatively Prime Number is : 7 */
 
